src/components/CryptoContext.tsx

```diff
@@ -0,0 +1,25 @@
+import { createContext, Dispatch, SetStateAction } from 'react';
+
+export interface CryptoContextType {
+  userCurrentPair: string[],
+  price: number,
+  isGoingUp: boolean
+}
+export const startInApp: CryptoContextType = {
+  userCurrentPair: ["BTC-USD"],
+  price: 0.00,
+  isGoingUp: true
+}
+
+export interface ContextState {
+  context: CryptoContextType,
+  setContext?: Dispatch<SetStateAction<CryptoContextType>>
+}
+export const startContext: ContextState = {
+  context: startInApp
+}
+
+// this initial context needs to match types with useState
+export const CryptoContext = createContext(startContext);
```

Comment:
Alright makes sense like we are referencing those states. It makes sense it has to wrap around the components.